### PR TITLE
IA for Power Automate tutorials

### PR DESCRIPTION
**Pull request by AlexJerabek**
The titles of the two Power Automate tutorials are too similar. This should address that concern.

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[1427071](https://github.com/OfficeDev/office-scripts-docs/commits/1427071ad70800e6d7f57b892f04c563caf64e3e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |
[docs/includes/power-automate-tutorial-prerequisites.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/includes/power-automate-tutorial-prerequisites.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-power-automate-manual.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/tutorials/excel-power-automate-manual.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-power-automate-trigger.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/tutorials/excel-power-automate-trigger.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C14%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202007142050557775-147%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=rSzNijT9UesIMBJK69iKycE9%2B2P%2FZrdgGTbMp0%2F9kBY%3D&st=2020-07-14T20%3A47%3A05Z&se=2020-08-14T20%3A52%3A05Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by o365devx** [comment]
Docs Build status updates of commit _[6e6295b](https://github.com/OfficeDev/office-scripts-docs/commits/6e6295b3f0eba94957971e0a168229d0e2baf6f5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/develop/power-automate-integration.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/develop/power-automate-integration.md) | :white_check_mark:Succeeded | |
[docs/includes/power-automate-tutorial-prerequisites.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/includes/power-automate-tutorial-prerequisites.md) | :white_check_mark:Succeeded | |
[docs/includes/tutorial-prerequisites.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/includes/tutorial-prerequisites.md) | :white_check_mark:Succeeded | |
[docs/toc.yml](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/toc.yml) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-power-automate-manual.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/tutorials/excel-power-automate-manual.md) | :white_check_mark:Succeeded | |
[docs/tutorials/excel-power-automate-trigger.md](https://github.com/OfficeDev/office-scripts-docs/blob/AlexJ-PAFacts/docs/tutorials/excel-power-automate-trigger.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C7%5C14%5C918d1c35-f3a4-631b-1edf-a1a2af22eee5%5CPullRequest%5C202007142328475357-147%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=E21dxP2bR0UbYAoNs32q3PAys2YKHn93gDHLzq2d%2BY8%3D&st=2020-07-14T23%3A25%3A08Z&se=2020-08-14T23%3A30%3A08Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/918d1c35-f3a4-631b-1edf-a1a2af22eee5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/tutorials/excel-power-automate-manual.md [hunk 1]
```diff
@@ -121,4 +121,4 @@ Your flow is now ready to be run through Power Automate. You can test it using t
 
 ## Next steps
 
-Complete the [Automatically run scripts with automated Power Automate flows](excel-power-automate-trigger.md) tutorial. It teaches you how to pass data from a workflow service to your Office Script.
+Complete the [Pass data to scripts in an automatically-run Power Automate flow](excel-power-automate-trigger.md) tutorial. It teaches you how to pass data from a workflow service to your Office Script and set the Power Automate flow to run based on events instead of manual button presses.
```
Comment (mscharlock) [inline docs/tutorials/excel-power-automate-manual.md:124]: Do you need to specify manual button presses here? Maybe just leave it at "run based on events"

File: docs/tutorials/excel-power-automate-trigger.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ---
-title: 'Automatically run scripts with automated Power Automate flows'
-description: 'A tutorial about running Office Scripts for Excel on the web through Power Automate by using an automatic external trigger (receiving mail through Outlook).'
-ms.date: 07/10/2020
+title: 'Pass data to scripts in an automatically-run Power Automate flow'
+description: 'A tutorial about running Office Scripts for Excel on the web through Power Automate by using an automatic external trigger (receiving mail through Outlook) and passing data from the previous flow step to the script.'
```
Comment (mscharlock) [inline docs/tutorials/excel-power-automate-trigger.md:3]: Is there a way to cut this down? It is just a really long description

**Comment by AlexJerabek** [comment]
Thanks for the suggestions @mscharlock! Updated accordingly.

**Pull request merged**